Sources/Sentry/include/SentrySpan+Private.h

```diff
@@ -0,0 +1,9 @@
+#import "SentrySpan.h"
+
+@interface
+SentrySpan ()
+
+@property (nonatomic) uint64_t startSystemTime;
+@property (nonatomic) uint64_t endSystemTime;
```

Comment:
Instead of adding these, changing the underlying start and end time to system time is better. Basically accomplishing https://github.com/getsentry/sentry-cocoa/issues/2155

Reply:
So I started on that change when I implemented concurrent transaction profiling, but it has far-reaching consequences. I'd prefer to leave it out of scope for this PR.

Reply:
I wouldn't do it in this PR either, but it would be great to accomplish https://github.com/getsentry/sentry-cocoa/issues/2155 before doing this change.

Reply:
Is there a pressing need for it? Because delaying this PR by that much time I don’t think is necessary. We need this PR’s capability to fix issues with dynamic sampling.

Reply:
> Is there a pressing need for it? Because delaying this PR by that much time I don’t think is necessary

No not pressing need. Just the right thing to do.

>  I believe fully resolving that issue will require backend and frontend changes as well.

Why do you think we need changes there? 

I'm okay with moving forward with this for now. It's an acceptable tradeoff.

Reply:
> Why do you think we need changes there?

In the backend, at the bare minimum we'll have to change units, and number types from float to integer. In the frontend we'll have to change how those are displayed, and this would be an addition to the NSDate information showing the calendar datetime an event occurred at, only displaying duration in the system time format. Then we'd have to do some user education on what the differences mean.

Reply:
+1 on this being an urgent change that needs to ship as soon as possible, we can decouple other changes from this